mlir/lib/Dialect/LLVMIR/IR/LLVMDialect.cpp

```diff
@@ -3108,35 +3102,27 @@ OpFoldResult LLVM::ZeroOp::fold(FoldAdaptor) {
 //===----------------------------------------------------------------------===//
 
 /// Compute the total number of elements in the given type, also taking into
-/// account nested types. Supported types are `VectorType`, `LLVMArrayType` and
-/// `LLVMFixedVectorType`. Everything else is treated as a scalar.
+/// account nested types. Supported types are `VectorType` and `LLVMArrayType`.
+/// Everything else is treated as a scalar.
```

Comment:
Should we maybe clarify that this yields funky results when working with a scalable `VectorType`?

Reply:
I added an assertion to exclude scalable vectors.